Cingulara/openrmf-api-save
# release

- putting the latest version tag in here
- Adding Jaeger Tracing Information
- adding auditing information
- prometheus exporting metrics
- updating the copyright notice
- updating the logging for API calls
- updating API descriptions
- adding copyright notice
- #28 allow deleting a group of checklists
- #28 allow deleting a group of checklists
- #28 allow deleting a group of checklists
- Fixing t he version number
- #30 allow a create/add on a system
- #29 allow deleting by passing in a system ID
- Naming the NATS client
- #25 update data with eventual consistency NATS
- #25 fixing the published message
- #25 fixing problem with the logic
- #25 allow updating the system information
- #25 initial system to checklist grouping
- #24 adding NATS options and reconnect
- updating to version 0.9
- fixing ENV vars to be all caps
- update NATS client and subject
- updating info to version 0.8
- RBAC updates with keycloak for v0.8
- Reorg for unit tests
- getting the delete to work correctly
- updating the Artifact model
- adding delete API call
- renaming openRMF ; adding delete function
- update for openRMF
- updating for version 0.6
- updating the version
- adding the system name for checklists
- Adding a health controller
- upload new returns the InternalId to send to Score
- #2 update to the Saved Artifact minus checklist data
- updating artifacts #1 #2, removing the id for InternalId on Updates
- #9 add remaining STIG types that are not PKI restricted
- updating Makefile and logging to console
- adding Makefile and updated time on creation
- removing old code
- updating types of STIGs
- Updating license to GPL3
- fixing naming of the api
- adding in passing NATS server url
-  Making #3 final for docker images and docs
- Dockerfile for #3 to finalize
- updating the controller update publish event
- updating the update call on the id and internal ID -- needs to be tested
- updating readme directions
- Adding NATS messaging to the save
- updating the swagger API title info
- Updating Artifact class with user and description.
- updating readme description
- adding an update path, removing all gone
- removing files not needed for CQRS
- fix the other mongodb connection string to localhost
- updating mongodb connection string to localhost
- fixing documentation for the calls
- fixing the loader name
- updating docs and classes for saving
- updating the save function for a string to use
- cleaning up code and api root call
